cost_rules/tag_client.py

```diff
@@ -0,0 +1,68 @@
+import logging
+import re
+
+
+LOG = logging.getLogger(__name__)
+LOG.setLevel(logging.INFO)
+
+# This is global so that it can be stubbed in test.
```

Comment:
doesn't the boto stubber handle stubbing non globals?  does this statement still apply?

Reply:
yeah, we still need to define this variable outside of the function below so that we can create a client and stubber before calling the function.